src/main/java/io/spokestack/spokestack/Profile.java

```diff
@@ -0,0 +1,48 @@
+package io.spokestack.spokestack;
+
+/**
+ * Convenience shortcuts to pre-built {@link PipelineProfile}s used to configure
+ * the {@link SpeechPipeline}.
+ */
+public enum Profile {
```

Comment:
creating an enumeration that has to know about every type of a thing solely for the purpose of trying to make clients independent of a string constant is a poor design
in general, components that are coupled to all instances of a thing horizontally should be avoided
further, if the client implements their own profile type, there is no way to integrate it into this enumeration, so the abstraction is lost anyway
I would remove this enum